src/Module/Template/Step/ProcessTwigLayouts.php

```diff
@@ -65,6 +66,10 @@ private function createTwig($templateDirectory)
         ]);
 
         if (file_exists($templateDirectory.'/twig.php')) {
+            /**
+             * @psalm-suppress UnresolvableInclude
+             * @var callable(Twig_Environment): void
```

Comment:
Here we type with `Twig_Environment` but in the documentation of this feature you use `Twig\Environment`, should I replace `Twig_Environment` by the namespaced class name ? (There shouldn't be a difference since it's an alias, but I'm not sure of potential side effects)

Reply:
Honestly go the way you prefer, I don't there should be side effects either.

Reply:
Since the `composer.json` refers `twig ~1.10` the namespace isn't provided in this version, I'll change the documentation